file_paths.md

```diff
@@ -132,124 +136,129 @@ user or group.
-        puppetres.dll                         # event log message resource dll
-        versions.txt                          # versions of components
-
-    C:\Program Files\Puppet Labs\Puppet\puppet
-        bin
-            puppet                            # ruby bin wrapper
-        lib
-            puppet.rb
-
-    C:\Program Files\Puppet Labs\Puppet\service
-        daemon.rb                             # windows service daemon
-
-    C:\Program Files\Puppet Labs\Puppet\sys
-        ruby
-            bin
-                ruby.exe
-                ssleay32.dll                  # openssl dll
-            include
-            lib
-            share
-
-        tools
-            bin
-                elevate.exe                   # Used to elevate interactive commands
+        plugins *                             # user installed plugins
+        etc *                                                                same
+            client.cfg
+            facts.yaml
+            server.cfg
+        var *                                                                same
+            log *
+                mcollective.log
+
+    C:\ProgramData\PuppetLabs\puppet
+        cache                                 # :vardir                      C:\ProgramData\PuppetLabs\puppet\var
+            bucket                            # :bucketdir
+            client_yaml                       # :clientyamldir
+            client_data                       # :client_datadir
+            clientbucket                      # :clientbucketdir
+            devices                           # :devicedir
+            facts.d                           # :pluginfactdest (pluginsync'ed)
+            lib                               # :libdir
+            facts                             # used to generate :factpath
+            puppet-module                     # :module_working_dir
+            reports                           # :reportdir
+            server_data                       # :server_datadir
+            state                             # :statedir
+            yaml                              # :yamldir
+        etc *                                 # :confdir                     same
+            auth.conf                         # :rest_authconfig
+            autosign.conf                     # :autosign
+            binder_config.yaml                # :binder_config
+            csr_attributes.yaml               # :csr_attributes
+            custom_trusted_oid_mapping.yaml   # :trusted_oid_mapping_file
+            device.conf                       # :deviceconfig
+            fileserver.conf                   # :fileserverconfig
+            puppet.conf *                     # :config
+            routes.yaml                       # :route_file
+            ssl                               # :ssldir
+        var *
+            log                               # :logdir                      same
```

Comment:
comment from https://github.com/joshcooper/puppet-specifications/commit/6fd4098708192fb8bcf68664c78adaca8f1a9203#commitcomment-10033811, var is created but log is not.